src/components/IntelligentTreeSelect.js

```diff
@@ -519,11 +521,12 @@ IntelligentTreeSelect.defaultProps = {
   multi: true,
   options: [],
   renderAsTree: true,
-  isMenuOpen: false,
+  isMenuOpen: undefined,
```

Comment:
Why change the default from `false` to `undefined`?

Reply:
You are right, it is not necessary to change it.